LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIOutpostManagement.uc

```diff
@@ -83,176 +94,347 @@ function OnRadioTowerUpgradeClicked(UIButton Button)
 
 simulated function InitScreen(XComPlayerController InitController, UIMovie InitMovie, optional name InitName)
 {
-    local int NextY;
-    local XComGameState_WorldRegion Region;
-    local XComGameState_LWOutpost Outpost;
-    local XComGameStateHistory History;
-	local string strRegionalInfo;
-    local string strResistanceMecs;
-	local string strJobDetail;
-	local XGParamTag kTag;
-	local XComGameState_WorldRegion_LWStrategyAI RegionalAIState;
 	local bool IntelProhibited, SupplyProhibited, RecruitProhibited;
-	local int k;
+	local int AvailableSpace, i, AdviserIconSize, AdviserBorderPadding, ScrollbarPadding;
 
- 	// Init UI
-	super.InitScreen(InitController, InitMovie, InitName);
+    local int NextX, NextY;
+	local XComGameState_LWOutpost Outpost;
+	local XComGameState_WorldRegion Region;
+    local XComGameStateHistory History;
 	
-    MainPanel = Spawn(class 'UIPanel', self);
-    MainPanel.InitPanel('ListContainer').SetPosition((Movie.UI_RES_X / 2) - (panelW / 2), panelY);
-    ListBG = Spawn(class'UIBGBox', MainPanel);
-	ListBG.LibID = class'UIUtilities_Controls'.const.MC_X2Background;
-	ListBG.InitBG('ListBG', 0, 0, panelW, panelH);
-    ListBG.bIsNavigable = false;
-
-    History = `XCOMHISTORY;
+	History = `XCOMHISTORY;
     Outpost = XComGameState_LWOutpost(History.GetGameStateForObjectID(OutpostRef.ObjectID));
     Region = XComGameState_WorldRegion(History.GetGameStateForObjectID(Outpost.Region.ObjectID));
-    ListTitle = Spawn(class'UIX2PanelHeader', MainPanel).InitPanelHeader('TitleHeader', m_strTitle, Region.GetDisplayName());
-	ListTitle.SetHeaderWidth(ListBG.Width - 20);
-    ListTitle.SetPosition(ListBG.X + 10, ListBG.Y + 10);
+	
+	DisplayTag = 'UIDisplay_Council';
+	CameraTag = 'UIDisplayCam_ResistanceScreen';
```

Comment:
These should probably be constants initialised in the `defaultproperties` block rather than here. Unless they truly are variables? Also, why are they being used now but not in the original LW2 code?